src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -366,7 +366,7 @@ export default {
 			}
 		},
 		trackFirstRowDisplayedLoans() {
-			this.trackDisplayedLoans(this.enableRelendingExp ? 'relending' : 'recommended', 1, this.firstRowLoans);
+			this.trackDisplayedLoans(this.enableRelendingExp && !this.enableRecommendedReplacementExp ? 'relending' : 'recommended', 1, this.enableRecommendedReplacementExp ? [this.featuredLoan] : this.firstRowLoans); // eslint-disable-line max-len
```

Comment:
We will be removing the relending experiment, so you could simplify this line now.